install.bash

```diff
@@ -115,11 +118,13 @@ fi
 echo "***** INSTALLING FIFTYONE *****"
 if [ ${DEV_INSTALL} = true ] || [ ${VOXEL51_INSTALL} = true ]; then
     echo "Performing dev install"
-    pip install -r requirements/dev.txt
-    pre-commit install
-    pip install -e .
+    pip install --no-deps -r requirements/dev.txt
```

Comment:
`--no-deps` avoids conflict resolution - versions are already pinned in the requirements files